features/step_definitions/custom_web_steps.rb

```diff
@@ -298,6 +298,7 @@ def matches_path?(url, path)
 
   toggle.click
   find(:xpath, link).click
+  wait_for_requests
```

Comment:
In our case, it always does. Because we only call this from the CMS page to click on 'Hide' using AJAX.

![image](https://user-images.githubusercontent.com/11318903/83519066-a4ecd900-a4db-11ea-940e-0f88f25b11c1.png)

https://github.com/3scale/porta/blob/cc144dd52b74ceae7b297c8cfaa836908a1baed3/features/old/cms/pages.feature#L1

https://github.com/3scale/porta/blob/cc144dd52b74ceae7b297c8cfaa836908a1baed3/features/old/cms/pages.feature#L51-L53

https://github.com/3scale/porta/blob/cc144dd52b74ceae7b297c8cfaa836908a1baed3/features/old/cms/pages.feature#L58-L75

However, I understand the question because the name of the step is very generic.
So perhaps I could do instead:
```suggestion
wait_for_requests if @javascript
```
WDYT?

Reply:
No need because even if we need javascript it does not mean AJAX
Also the tag is on the scenario not on the step and we want it on the step